src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -153,9 +151,9 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine($"#Keep Alive interval (set in seconds). Default: 0 (no keep alive).");
             builder.AppendLine($"#keepalive=0");
             builder.AppendLine($"#Use HTTPS protocol on the API. Default is { false }.");
-            builder.AppendLine($"#usehttps={DefaultCertificateFileName}");
+            builder.AppendLine($"#usehttps={ false }");
```

Comment:
Maybe we don't need interpolated hardcoded value and just use the value as we do for example as we do on line 152

Reply:
done